server-common/src/main/java/org/apache/kafka/server/common/TestFeatureVersion.java

```diff
@@ -32,6 +32,9 @@ public enum TestFeatureVersion implements FeatureVersion {
 
     public static final String FEATURE_NAME = "test.feature.version";
 
+    public static final TestFeatureVersion LATEST_PRODUCTION =
+        MetadataVersion.latestProduction() == MetadataVersion.latestTesting() ? TEST_2 : TEST_1;
```

Comment:
The comment above says "TEST_2 is not yet released ". We probably want to adjust it to make it more accurate.